sample-app/src/components/InputDropdown.tsx

```diff
@@ -49,6 +50,7 @@ export default function InputDropdown({
   screenReaderInstructions,
   screenReaderInstructionsId,
   screenReaderText,
+  onlySubmitOnOption,
```

Comment:
Can you explain the purpose of this option?

Reply:
It controls whether you want to allow users to submit only when they've selected a dropdown option, or if they can submit whatever they've typed.

In the case of the SearchBar, they should be allowed to submit anything, but for FilterSearch, they can only select a filter from the dropdown and nothing should happen if they press Enter on their own text without focusing on a dropdown option.

Reply:
Got it. What do you think of the name`onlyAllowDropdownOptionSubmissions`instead?

Reply:
That's more clear, so I changed it to that